src/Engine/ProtoCore/Utils/CoreUtils.cs

```diff
@@ -254,6 +254,20 @@ public static bool TryGetOperator(string methodName, out Operator op)
             return System.Enum.TryParse(realMethodName, out op);
         }
 
+        public static bool TryGetUnaryOperator(string methodName, out UnaryOperator op)
+        {
+            Validity.Assert(null != methodName);
+            if (!methodName.StartsWith(DSASM.Constants.kInternalNamePrefix))
```

Comment:
couldn't we use `CoreUtils.IsInternalMethod(methodName)` instead here ?

Reply:
Yes,  we could. I just copied the code from the function above for normal operators.